app/cmd/preview.go

```diff
@@ -416,6 +416,7 @@ preview and return/open the preview URL when it is complete.
 				previewCmdError(fmt.Sprintf("%v", err), tmpZipFile)
 				return
 			}
+			//previewer.addPathsFromConfigFiles()
```

Comment:
Please add information regarding when this can be removed.

Reply:
removed